packages/eas-cli/src/credentials/android/actions/RemoveGoogleServiceAccountKey.ts

```diff
@@ -0,0 +1,47 @@
+import { GoogleServiceAccountKeyFragment } from '../../../graphql/generated';
+import Log from '../../../log';
+import { confirmAsync } from '../../../prompts';
+import { Account } from '../../../user/Account';
+import { Context } from '../../context';
+import { selectGoogleServiceAccountKeyAsync } from '../utils/googleServiceAccountKey';
+
+export class SelectAndRemoveGoogleServiceAccountKey {
+  constructor(private account: Account) {}
+
+  async runAsync(ctx: Context): Promise<void> {
```

Comment:
Does running this action make sense in non-interactive mode? 
Maybe we should add a guard like this https://github.com/expo/eas-cli/pull/658/files#diff-4b3d674e01626e2dd920fba191efeec8e73653b2576b86ba96d5ac4f02393ffaR31

Reply:
guards like that should be needed only if action can be run in a context other than credentials manager, `eas credentials` is always interactive.

Reply:
@wkozyra95 How can we guarantee that no one will ever use this in another context?

Reply:
I've updated this to add the `non-interactive` guard at the beginning for fast failure, in case this is called in the wrong context